spec/features/cart_view_spec.rb

```diff
@@ -63,8 +64,58 @@
         expect(page).to have_content(8.42)
       end
     end
+
+    context 'when a user checks out' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.